pilot/pkg/labeltemp/label.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package label
+
+// TODO temp for testing until I move this into istio/api
```

Comment:
I think pkg.model may also be ok - but I remember there was some big move and process for labels..

Reply:
Where would I find an example of labels being defined in istio/istio
API PR: https://github.com/istio/api/pull/1546

Reply:
I think most (all?) things are currently defined as annotations today. If we start standardizing labels, we would want to follow the annotation pattern in istio/api. But I do think these should be labels instead of annotations ... for selection (especially network).